components/odh-notebook-controller/controllers/notebook_oauth.go

```diff
@@ -104,6 +107,70 @@ func (r *OpenshiftNotebookReconciler) ReconcileOAuthServiceAccount(notebook *nbv
 			log.Error(err, "Unable to fetch the Service Account")
 			return err
 		}
+
+		nbrole := &rbacv1.Role{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      fmt.Sprintf("%s-view", notebook.Name),
+				Namespace: notebook.Namespace,
+			},
+			Rules: []rbacv1.PolicyRule{
+				{
+					Verbs:     []string{"get", "watch", "list"},
+					APIGroups: []string{""},
+					Resources: []string{"configmaps", "pods"},
```

Comment:
The roles added here are limited to pods and configmaps, are there other resources a user might find convenient to view from workbench terminal?
Also the `oc cluster-info` will fail here as that will require access to a different namespace, do we need it?
cc @harshad16